front/public/locales/pt/stdcm-help-section.json

```diff
@@ -1 +1,326 @@
-{}
+{
+  "asu": [
+    {
+      "title": "Nome da aplicação",
+      "value": "OSRD"
+    },
+    {
+      "title": "Assistência",
+      "value": "Suporte para aplicações ASU"
+    },
+    {
+      "title": "Telefone",
+      "value": "70 70 70 (ou 09 72 72 12 70)"
+    },
+    {
+      "title": "Código da Aplicação (AVAYA)",
+      "value": "176"
+    }
+  ],
+  "asuLink": "Ligação para suporte ASU",
+  "backToIndex": "índice da ajuda",
+  "help": "Ajuda",
+  "sections": {
+    "application": {
+      "content": [
+        {
+          "type": "texto",
```

Comment:
The type of the block should remain in english, otherwise it can't be interpreted in our code
```suggestion
          "type": "text",
```

<img width="517" height="953" alt="image" src="https://github.com/user-attachments/assets/85c82710-2401-42bf-bd97-0c30cbcbfef2" />